tests/docker-compose.yml

```diff
@@ -41,10 +41,13 @@ services:
             - ENABLE_IPV6=true
             - DEV_MODE=true
         volumes:
-            - ..:/elabftw
+            - ./..:/elabftw
```

Comment:
is that different from simply `../` ?

Reply:
I don't know.
Initially I wanted to bind mount the files but I couldn't make it work. So now there is the new dockerfile and files are copied and the volumes lines in compose.yml are removed with sed.
So this is reverted to `..`